h/templates/panels/navbar.html.jinja2

```diff
@@ -13,6 +13,20 @@
             id="search-bar"
             action="{{ search_url }}">
         {{ svg_icon('search', 'search-bar__icon') }}
+        <div class="search-bar__lozenges">
+          {% if opts['search_username'] %}
```

Comment:
I would suggest we extract this into a helper `macro()` and add a comment to the effect that it is a static version of the UI generated by `LozengeController`